src/components/BookComponent.jsx

```diff
@@ -0,0 +1,47 @@
+import './Component.css';
+import { useState, useEffect } from 'react';
+
+const BookComponent = () => {
+  const [book, setBook] = useState([]);
+
+  const bookData = [
+    { title: ' The Hunger games', author: 'Suzanne Collins', id: 1 },
+    { title: ' Dune', author: 'Frank Herbert', id: 2 },
+    { title: ' Capital in the Twenty-First Century', author: 'Suzanne Collins', id: 3 },
+  ];
+
+  useEffect(() => {
+    setBook(bookData);
+  }, [book]);
+  return (
+    <>
+      <div className="bookStore">
+        {book ? bookData.map((book) => (
+          <div key={book.id}>
+            <p className="title">{book.title}</p>
+            <p className="bookAuthor">{book.author}</p>
+            <div className="buttons">
+              <button type="button" className="removeBtn">Remove</button>
+              <button type="button" className="removeBtn">Comments</button>
+              <button type="button" className="removeBtn">Edit</button>
+            </div>
+          </div>
+        )) : null}
+      </div>
+      <div className="formContainer">
+        <form>
+          <input type="text" placeholder="Book title" className="bookTitle" />
+          <select name="books" id="books" className="selectCategory">
+            <option value="category">category</option>
+            <option value="fiction">fiction</option>
+            <option value="adventure">adventure</option>
+            <option value="entertainment">entertainment</option>
+          </select>
+          <button type="submit" className="submitButton">ADD BOOK</button>
+        </form>
+      </div>
+    </>
+  );
+};
+
+export default BookComponent;
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, according to the requirements:

![image](https://github.com/ismayito/Bookstore-react-app/assets/77069011/b5ba9fd5-afd3-4d79-ac3c-2d543bc77e4d)

Please consider creating components for an individual book and the form to create new books. You can extract them from this code:


```js
// Book.js
const Book = ({id, title, author}) => {
  return (
    <div>
      <p className="title">{title}</p>
      <p className="bookAuthor">{author}</p>
      <div className="buttons">
        <button type="button" className="removeBtn">Remove</button>
        <button type="button" className="removeBtn">Comments</button>
        <button type="button" className="removeBtn">Edit</button>
      </div>
    </div>
  )
}
```

```js
// BookForm.js
const BookForm = () => {
  return (
    <div className="formContainer">
      <form>
        <input type="text" placeholder="Book title" className="bookTitle" />
        <select name="books" id="books" className="selectCategory">
          <option value="category">category</option>
          <option value="fiction">fiction</option>
          <option value="adventure">adventure</option>
          <option value="entertainment">entertainment</option>
        </select>
        <button type="submit" className="submitButton">ADD BOOK</button>
      </form>
    </div>
  )
}
```

Then, simply use these components in your code:

```suggestion
const BookComponent = () => {
  const [book, setBook] = useState([]);

  const bookData = [
    { title: ' The Hunger games', author: 'Suzanne Collins', id: 1 },
    { title: ' Dune', author: 'Frank Herbert', id: 2 },
    { title: ' Capital in the Twenty-First Century', author: 'Suzanne Collins', id: 3 },
  ];

  useEffect(() => {
    setBook(bookData);
  }, [book]);
  return (
    <>
      <div className="bookStore">
        {book ? bookData.map((book) => (
          <Book key={book.id} id={book.id} title={book.title} author={book.author}/>








        )) : null}
      </div>
      <BookForm />











    </>
  );
};

export default BookComponent;
```

Now, your Booklist looks cleaner and easier to manage! 🤓 🔧